SPEC.md

```diff
@@ -52,7 +54,9 @@ The request-body must has the `notifications` array. There is the parameter tabl
 |-----------------|------------|-----------------------------------------|--------|-------|----------------|
 |token            |string array|device tokens                            |o       |       |                |
 |platform         |int         |platform(iOS,Android)                    |o       |       |1=iOS, 2=Android|
+|title            |string      |title for notification                   |o       |       |only iOS        |
 |message          |string      |message for notification                 |o       |       |                |
+|subtitle         |string      |subtitle for notification                |o       |       |only iOS        |
```

Comment:
yes, title and subtitle are optional values.
If they weren't set, only `message` will be displayed.

As below, these values are optional.
https://github.com/RobotsAndPencils/buford/blob/1589c179b764ddceb204a439d9bf366f04c55f9b/payload/aps.go#L37